docs/metrics.md

```diff
@@ -0,0 +1,120 @@
+# Metrics
+
+Test metrics are sent to 3 different places:
+1. reported properies
+2. test telemetry message bodies
+3. Azure Monitor
+
+Not all metrics are sent to all destinations.
+
+| group | reported properties | test telemetry | Azure Monitor |
+| Session Metrics | X | X | - |
+| Test Metrics | X | X | X |
+| System Health Metrics | - | X | X |
+
+## Session Metrics
+
+Some session metrics are recorded at the beginning of the run and others are recorded throughout the run.
+They record details on the progress of the current run.
+
+```json
+  {
+    "reported": {
+      "thief": {
+        "sessionMetrics": {
+          "exitReason": "Main thread raised <class KeyboardInterrupt>",
+          "latestUpdateTimeUtc": "2020-12-14T21:14:57.447061",
+          "pairingId": "176e7469-9755-4a7a-af74-85f735f0f296",
+          "runEndUtc": "2020-12-14T21:14:57.441870+00:00",
+          "runStartUtc": "2020-12-14T21:13:22.430280+00:00",
+          "runState": "interrupted",
+          "runTime": "0:01:35.016760"
+        }
+      }
+    }
+  }
+```
+
+| field | format | meaning |
+| - | - | - |
+| `runState` | string | State of the current run.  One of `waiting`, `running`, `failed`, `complete`, or `interrupted` |
```

Comment:
Any reason these values can't be capitalized (e.g. Waiting, Running)

Reply:
Sure.  Will change later